pennylane/devices/default_tensor.py

```diff
@@ -205,7 +246,11 @@ def circuit(num_qubits):
             We can provide additional keyword arguments to the device to customize the simulation. These are passed to the ``quimb`` backend.
```

Comment:
You can make a separate toggle section with a custom title like [so](https://github.com/PennyLaneAI/pennylane/blob/master/pennylane/gradients/pulse_gradient.py#L475) My idea was to have a separate "usage details"-kind section for MPS and TN methods

Reply:
I like the idea. I thought of doing something similar at first. Still, here I followed the [development guide for new classes](https://docs.pennylane.ai/en/stable/development/guide/documentation.html#classes) in PennyLane, which specifies one unique expandable section called 'Usage Details' for the device. I have no problems in creating 2 different sections if that's allowed @trbromley

Reply:
Green light: I created 2 separate expandable sections!